File: research/stellar/examples/javascript/account/check-account.js
const StellarSdk = require('stellar-sdk')
const server = new StellarSdk.Server('https://horizon-testnet.stellar.org')

var argv = require('yargs')
    .usage('Usage: $0 --address [string]')
    .demandOption(['address'])
    .argv;

async function checkAccountBalance (address) {
  account = await server.loadAccount(address)
  account.balances.forEach(balance => {
    console.log(balance)
  })
}

checkAccountBalance(argv.address)

File: research/stellar/examples/python/account/check-account.py
# pylint: disable=no-value-for-parameter
from stellar_sdk import Server
import click

@click.command()
@click.option('--address', help='Address to check', required=True)

def check_account(address):
  
  server = Server(horizon_url="https://horizon-testnet.stellar.org")
  response= server.accounts().account_id(address).call()
  print(response['balances'])

if __name__ == '__main__':
  check_account()